pysoa/test/stub_service.py

```diff
@@ -378,7 +379,10 @@ def __enter__(self):
         mock_action = self._MockAction(name='{}.{}'.format(self.service, self.action))
 
         if self.body or self.errors:
-            mock_action.side_effect = lambda _: ActionResponse(self.action, errors=self.errors, body=self.body)
+            mock_action.return_value = ActionResponse(self.action, errors=self.errors, body=self.body)
+
+        if self.side_effect:
```

Comment:
Is it a breaking change to any existing tests using `side_effect`?

Reply:
nope, cause they are both (old and new) evaluated before the mock is created and returned to the test itself